worlds/sc2/docs/setup_fr.md

```diff
@@ -0,0 +1,191 @@
+# Guide d'installation du *StarCraft 2 Randomizer*
+
+Ce guide contient les instructions pour installer et dépanner le client de *StarCraft 2 Archipelago*, ainsi que des 
+indications pour obtenir un fichier de configuration de *Starcraft 2 Archipelago* et comment modifier ce dernier.
+
+## Logiciels requis
+
+- [*StarCraft 2*](https://starcraft2.com/en-us/)
+- [La version la plus récente d'Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases)
+
+## Comment est-ce que j'installe ce *randomizer*?
+
+1. Installer *StarCraft 2* et Archipelago en suivant les instructions indiquées dans les liens précédents. Le client de 
+*StarCraft 2 Archipelago* est téléchargé par le programme d'installation d'Archipelago.
+   - Les utilisateurs de Linux devraient aussi suivre les instructions qui se retrouvent à la fin de cette page 
+(["Exécuter sous Linux"](#exécuter-sous-linux)).
+2. Exécuter `ArchipelagoStarcraft2Client.exe`.
+   - Uniquement pour cette étape, les utilisateurs de macOS devraient plutôt suivre les instructions qui se trouvent à 
+["Exécuter sous macOS"](#exécuter-sous-macos).
+3. Dans le client de *StarCraft 2 Archipelago*, écrire la commande `/download_data`. Cette commande va lancer 
+l'installation des *Maps* et des *Data files* qui sont nécessaires pour jouer à *StarCraft 2 Archipelago*.
```

Comment:
cartes? (cela n'est pas important, juste des suggestions)

Reply:
I decided to side-step the problem by replacing both english terms by "fichiers". Good with you?

Reply:
To be perfectly honest it sounds fine!

Reply:
Honestly it's also fairly general and understandable too so that makes lots of sense.

Reply:
Je suis en désaccord. Le dossier sur le disque s'appellera toujours "Maps", donc l'appeller comme tel rend ça plus clair si la personne a besoi de vérifier si ça  a marché, le path, etc